docs/docs/secrets-managers.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: secrets-managers
+sidebar_label: Secrets Managers
```

Comment:
My 2 cents:
I originally named to feature Credentials Management but in the code I used Secrets Manager a lot. Mostly because I worked with AWS Secrets Manager in the past.
From code standpoint it would be easier to use Secrets Manager and to rename discovery ticket in JIRA.

@andreaklmn WDYT?

Reply:
I think we should name it in a way that makes sense for our customers and our overall messaging. If both are equivalent, I agree with what you said and we should pick the simpler choice for us. But only if they're equivalent :D

Reply:
I guess for me the question is: are we managing all types of sensitive data, including passwords, API keys, encryption keys, certificates, and other types of confidential information - if yes, then it's "secrets management"

If we are only managing credentials, I would probably stick to credentials management..? Unless we are planning on managing other "secrets" as part of this solution

My 2 cents!

Reply:
I just saw that the Vault website here uses the 2 terms somewhat interchangeably: https://www.vaultproject.io/
Under "automate credentials rotation" - so I'm down for using secrets management!

Reply:
For anyone stumbling on this PR: We have discussed this and agreed to use Secrets Manager.
Just to put my answer here as well, we are able to manage sensitive data.